tests/test_nitpicking.py

```diff
@@ -193,7 +193,9 @@ def test_cidr_whitelist_rules(
     ].split(", ")
 
 
-def test_code_workspace_file(tmp_path: Path, cloned_template: Path):
+def test_code_workspace_file(
+    tmp_path: Path, cloned_template: Path, supported_odoo_version: float
+):
     """The file is generated as expected."""
     copy(
         str(cloned_template),
```

Comment:
I see you added the `supported_odoo_version` fixture.

The, for Copier to use it, you need to pass it here when calling `copy()`.

Se the test above: https://github.com/Tecnativa/doodba-copier-template/pull/115/commits/6966c8c4871538bbe46d4ada23d0548714c014de#diff-b56a11556ba1e38ee41432a50540bb5bR148

Reply:
Changes made. Pls check if that was what you meant.